src/components/panes/ObjectivePane.vue

```diff
@@ -172,6 +172,32 @@ export default {
       },
     },
   },
+
+  methods: {
+    isMemberOfChild() {
+      const { department, organization, id: parentId } = this.activeObjective.parent;
+
+      if (!organization && !department) {
+        const departmentsWithUser = this.departments.filter(
+          (dep) =>
+            dep.organization.id === parentId &&
+            dep.team.map(({ id }) => id).includes(this.user.id)
+        );
+        return departmentsWithUser.length > 0;
+      }
```

Comment:
Kan gjøres mer effektiv ved å bytte ut `filter` med `some`:

```js
if (!organization && !department) {
  return this.departments.some(
    (dep) =>
      dep.organization.id === parentId &&
      dep.team.map(({ id }) => id).includes(this.user.id)
  );
}
```

`some` avbryter ved første treff i stedet for å gå gjennom hele lista som `filter` gjør.